coderd/database/queries/users.sql

```diff
@@ -22,11 +22,14 @@ WHERE
 LIMIT
 	1;
 
--- name: GetUserCount :one
+-- name: GetActualUserCount :one
+-- Actual user count refers to the count of all users except the system user
 SELECT
 	COUNT(*)
 FROM
-	users;
+	users
+WHERE
+    id != '11111111-1111-1111-1111-111111111111';
```

Comment:
I don't believe this is a valid uuid, it needs some certain bits set. See https://github.com/google/uuid/blob/master/version4.go#L53

Reply:
TIL

Reply:
Thank you! Updated it to `c0de2b07-0000-4000-A000-000000000000`.